scripts/jest/createCacheKeyFunction.js

```diff
@@ -7,34 +7,34 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 
-var crypto = require('crypto');
-var fs = require('fs');
-var path = require('path');
+ 'use strict';
+
+const crypto = require('crypto');
+const fs = require('fs');
+const path = require('path');
 
 function buildCacheKey(files, base) {
-  return files.reduce(function(src, fileName) {
-    return src + fs.readFileSync(fileName);
-  }, base);
+  return files.reduce(
+    (src, fileName) => src + fs.readFileSync(fileName),
+    base
+  );
 }
 
-var transformRoot = path.join(__dirname, '..');
-var cacheKeyFiles = [
+const transformRoot = path.join(__dirname, '..');
```

Comment:
I think const worked on 0.10 but only with `"use strict";` on top. It doesn't matter, jest doesn't work with anything pre node 4 :)